package-lock.json

```diff
@@ -1,21 +1,27 @@
 {
   "name": "kbar",
-  "version": "0.1.6",
+  "version": "0.1.0-beta.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "kbar",
-      "version": "0.1.6",
+      "version": "0.1.0-beta.1",
       "license": "MIT",
       "devDependencies": {
+        "@reach/portal": "^0.16.0",
```

Comment:
Anything that's needed for the lib to run and not just for the demo should probably be in `dependencies`.

Reply:
Ah that makes a lot of sense – everyone should have `react`/`react-dom` as a dependency themselves but not necessarily `@reach/portal`, `fast-equals`, and `match-sorter`. Seems like it doesn't _really_ seem to matter but as a formality should be properly organized